packages/webpack/src/runtime/assemble.js

```diff
@@ -24,29 +75,22 @@ const assembleRuntime = (KEY, runtimeModules) => {
     let sourceString
     if (file) {
       sourceString = readFileSync(file, 'utf-8')
-      sourceString = removeMultilineComments(sourceString)
+    }
+    if (rawSource) {
+      sourceString = rawSource
     }
     if (data) {
       sourceString = JSON.stringify(data)
     }
     if (json) {
       sourceString = `LAVAMOAT['${name}'] = (${sourceString});`
-    }
-    if (rawSource) {
-      sourceString = rawSource
-    }
-    if (shimRequire) {
-      sourceString = readFileSync(require.resolve(shimRequire), 'utf-8')
-    }
-    if ((rawSource || shimRequire) && sourceString) {
-      sourceString = removeMultilineComments(sourceString)
-      sourceString = `;(()=>{
-        const module = {exports: {}};
-        const exports = module.exports;
-          ${sourceString}
-        ;
-        LAVAMOAT['${name}'] = module.exports;
-      })();`
+    } else {
```

Comment:
With the new implementation of removing comments that actually works correctly I needed to prevent it from attempting to parse JSON, so this rearrangement of logic here was necessary.